### PR TITLE
Add sharer component

### PR DESCRIPTION
**Pull request by harrygr**

### CODE REVIEW

File: src/components/Sharer.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { Button, Form } from 'semantic-ui-react';
+
+interface Props {
+  url: string;
+  text: string;
+  title: string;
+}
+
+export const Sharer = (props: Props) => {
+  const share = navigator.share;
```
Comment (Jarrydcrawford) [inline src/components/Sharer.tsx]: I wanna see this working!

Comment (harrygr) [reply src/components/Sharer.tsx]: yeah, I can't test it unless deployed

Comment (harrygr) [reply src/components/Sharer.tsx]: Seems like it's only supported by Chrome 61 on Android 😞

https://caniuse.com/#feat=web-share

**Comment by harrygr** [comment]
Ollie has added some designs that call for a button to trigger copying the link with a modal to inidicate that it's happened. Needs a bit more work

File: yarn.lock [hunk 2]
```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (harrygr) [inline yarn.lock]: I notice all the integrity hashes got removed. Is this because my version of yarn is old?

Comment (Jarrydcrawford) [reply yarn.lock]: Hrm must be. No big deal, it'll get added in a future commit.

Comment (harrygr) [reply yarn.lock]: It looks like it. Just updated and the hashes are back!

File: src/components/Sharer.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  if (share) {
+    return <Button onClick={() => share(props)}>Share vote link</Button>;
+  }
+
+  return (
+    <Button
+      onClick={() => {
+        Clipboard.copy(props.url);
+        alert(
```
Comment (Jarrydcrawford) [inline src/components/Sharer.tsx]: Would much prefer an inline message solution but this is fine for now 🙏

Comment (harrygr) [reply src/components/Sharer.tsx]: yeah, thought about using the modal. Inline might be nicer, but think I'll add a toast message implementation at some point that can be reused in other places.

**Pull request merged**